src/utils/utils.cairo

```diff
@@ -51,17 +51,17 @@ namespace Helpers {
             assert is_sequence_32_bytes_or_less = 1;
         }
 
-        let is_sequence_15_bytes_or_less = is_le(i, 15);
+        let is_sequence_16_bytes_or_less = is_le(i, 16);
 
-        // 1 - 15 bytes
-        if (is_sequence_15_bytes_or_less == TRUE) {
+        // 1 - 16 bytes
+        if (is_sequence_16_bytes_or_less == TRUE) {
```

Comment:
I'm still unsure of how big are the benefits of using `FALSE` always, but this is inline with our recent optimisations:
`if (is_sequence_16_bytes_or_less != FALSE) {`

Reply:
we miss a simple benchmark I do agree